docs/architecture/modern-web-apps-azure/common-web-application-architectures.md

```diff
@@ -93,8 +93,7 @@ The simplest approach to scaling a web application in Azure is to configure scal
 
```

Comment:
line 37 - remove extra comma in "This follows the separation of concerns principle, and can"